.travis.yml

```diff
@@ -6,9 +6,15 @@ x-ccache-setup-steps: &ccache-setup-steps
 
 os: linux
 language: cpp
+# Currently this file can only support one PYTHON_VERSION.
+# To experiment with Python 3, comment out Python 2.7 and uncomment one of the Python 3 versions.
```

Comment:
IIUC, with this change, Python 2.7 is already commented out, so this comment needs to be updated?